public/js/cita.json

```diff
@@ -4230,7 +4230,7 @@
     "booking_location_id": "fdcde285-37fd-4858-b00a-4ee31b4488b1"
   },
   {
-    "title": "Blaby",
+    "title": "Joint Service Shop in Blaby",
     "address": "Joint Service Shop\n10 Forge Corner\nBlaby\nLeicester\nLeicestershire\nLE8 4FZ",
```

Comment:
Do we want to have a duplication of "Joint Service Shop" in the title and address?